### PR TITLE
Set en_US.UTF-8 as default locale

### PR DESCRIPTION
**Pull request by rlvoyer**
This seems like the right thing to do to ensure reasonable defaults WRT character sets / encoding in our marathon apps.

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -37,3 +37,7 @@ CMD ["run"]
 
 # LABEL must be last for proper base image discoverability
```
Comment (znep) [inline base/Dockerfile]: # LABEL must be last for proper base image discoverability

Comment (rlvoyer) [inline base/Dockerfile]: ☝️

File: base/Dockerfile [hunk 2]
```diff
@@ -37,3 +37,7 @@ CMD ["run"]
...
 LABEL repository.socrata/base=""
+
+# Add locale profiles and default to en_US.UTF-8
+RUN locale-gen en_US.UTF-8
```
Comment (znep) [inline base/Dockerfile]: Should we use C.UTF-8 instead?

Comment (znep) [inline base/Dockerfile]: Hrm maybe that isn't in 14.04.

Comment (rlvoyer) [inline base/Dockerfile]: That sounds right (more language/environment agnostic). Confirming that it exists in 14.04.

**Pull request merged**